openseadragon-svg-overlay.js

```diff
@@ -2,15 +2,19 @@
 
 (function() {
 
-    if (!window.OpenSeadragon) {
-        console.error('[openseadragon-svg-overlay] requires OpenSeadragon');
-        return;
+    $ = window.OpenSeadragon;
```

Comment:
I've just discovered that this line causes a conflict with jQuery because it's missing `var`... I've fixed.